blocks/accordion/accordion.js

```diff
@@ -0,0 +1,62 @@
+import { createTag, getMetadata } from '../../scripts/scripts.js';
+
+const selectors = {
+  accordion: 'accordion',
+  accordionItem: 'accordion-item',
+  accordionItemHeader: 'accordion-item-header',
+  accordionItemContent: 'accordion-item-content',
+  accordionItemOpen: 'accordion-item-open',
+  arrow: 'accordion-arrow-down',
+};
+
+const accordionType = getMetadata('accordion-type') || 'single';
```

Comment:
Why did you choose metadata to control the `accordion-type` over section metadata or a custom class name on the block?

Reply:
@arumsey 
I think that metadata  represents  configuration of the block. I use extra css classes if I need multiple version of view (as in my PR different styles for purple/white/bone template).  In case when we need to  change block behavior (that handled by JS) I prefer to use metadata instead of using additional class that don't do anything with styles and just effect behavior in JS.
what do you think?

Regarding "metadata" or "section metadata" I don't know the differences. From one perspective "metadata" is more general used to apply changes on whole page, "section metadata" related to section scope. With accordion thats tricky question, I don't want to have relation to section but in the same time page can contain multiple accordions with different behavior, in that case   "section metadata" more applicable here. 
Maybe we should handle both cases, WDYT?

Reply:
I use section metadata now, in this case multiple accordions on same page can be configured differently